relayer/chains/solana/listener.go

```diff
@@ -28,19 +29,222 @@ func (p *Provider) Listener(ctx context.Context, lastSavedHeight uint64, blockIn
 	// 	p.log.Error("failed to init xcall", zap.Error(err))
 	// }
 
+	// return nil
+
 	if err := p.SendMessage(ctx, &relayertypes.Message{
```

Comment:
Is this for test purpose or actually required ?

Reply:
uhh yeah, since solana cli does not have feature to send messages, this one is for testing purpose and its just temporary.